week07/my-todo-list/src/pages/HomePage.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import "../App.css";
+import CustomDate from "../CustomDate";
+
+const HomePage = () => {
+  const quote_url = "https://api.quotable.io/random";
+  const [quote, setQuote] = useState("");
+  const [author, setAuthor] = useState("");
+
+  useEffect(() => {
+    let isMounted = true;
+    async function fetchData() {
+      const response = await fetch(quote_url);
+      const data = await response.json();
+      if (isMounted) {
+        setQuote(data.content);
+        setAuthor(data.author);
+      }
+    }
```

Comment:
[x] 3. 로딩 시간 - 명언 api
```suggestion
     async function fetchData() {
      setIsLoading(true);
      const response = await fetch(quote_url);
      const data = await response.json();
      if (isMounted) {
        setQuote(data.content);
        setAuthor(data.author);
        setIsLoading(false);
      }
    }
```